modules/loadbalancer/manifests/maintenance.pp

```diff
@@ -0,0 +1,18 @@
+# == Class: loadbalancer::maintenance
+#
+# Sets up a machine so that the loadbalancer can be put
+# into maintenance mode.
+#
+class loadbalancer::maintenance {
+  file { '/usr/share/nginx/www':
+    ensure  => directory,
+    mode    => '0755',
+    owner   => 'deploy',
+    group   => 'deploy',
+    require => Package['nginx'],
```

Comment:
That package resource actually ensures nginx is absent. Class['nginx::package'] might be a better fit